en/docs/get-started/user-management-architecture.md

```diff
@@ -1,11 +1,9 @@
 # User Management Architecture
 
 User management functionality is provided by default in all WSO2
-Carbon-based products and is configured in the
-`         <PRODUCT_HOME>/repository/conf/user-mgt.xml        ` file. The
-following documentation introduces the main concepts in User Management,
-such as users, roles, permissions, user stores etc. and how they are
-used in WSO2 products.
+Carbon-based products. The following documentation introduces the main
+concepts in User Management, such as users, roles, permissions, user
+stores etc. and how they are used in WSO2 products.
```

Comment:
should change to "user stores, etc., and how they are used in WSO2 products"

Note the comma after "user stores" and the comma after "etc.,"